.changeset/tender-dots-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mysten/zklogin': minor
+---
+
+add support for verify zklogin personal msg
```

Comment:
could update this to just say verify signatures since you added transaction block verification too